lib/takeaway.rb

```diff
@@ -0,0 +1,19 @@
+# Allows takeaway to checkout current order and send confirmation text message using Twilio API
+require_relative 'send_sms' #twilio-ruby code hidden in .gitignore 
+
+class Takeaway
+
+  def initialize(order = Order.new)
+    @message = Message.new
```

Comment:
I'd suggest using the same approach as you have for the Order class here and making use of dependency injection.  That would decrease the coupling between your objects and make testing easier.